internal/common/metrics.go

```diff
@@ -0,0 +1,173 @@
+package common
```

Comment:
Not sure if it makes sense, but I thought would be a good idea to have 2 versions of the methods: one that extracts the data from the cluster/namespace and convert to a flat JSON structure, and another that gets this JSON and print as human-readable text to the console.  
This way would be simple to reuse this data on a web page or a CSV/Excel report.